src/InfRcTransport.h

```diff
@@ -351,6 +374,7 @@ class InfRcTransport : public Transport {
     int          clientSetupSocket; // UDP socket for outgoing setup requests
     int          clientPort;        // Port number associated with
                                     // clientSetupSocket
+    int          gidIndex;          // Index for GID
```

Comment:
I suggest amending this comment to reflect that you are speaking about the RoCE GID index.  The comment here seems to be inane in that the variable name already states the content of the comment.

Reply:
I'll update comment, and when we make a RoCE toggle, with make sure this is zero when we're not using RoCE